templates/_components/_lister.twig

```diff
@@ -0,0 +1,109 @@
+{# _components/_lister #}
+
+{#
+  Lister
+  Generate a list of entry views
+  
+  @param  entries          {array}    List of Elements
+  @param  kind             {string}   Name of view type that you want to use for each list item eg: 'teaser' | 'title' etc.
+  @param  class            {string}   Class names for list wrapper element
+  @param  list_class       {string}   Class names for <ul> tag element
+  @param  list_attributes  {array}    Attributes for <ul> tag element
+  @param  item_class       {string}   Class names for <li> tag element
+
+
+
+  Configuration
+  =============
+  There is a configurable option available which help save
+  some repetition when using the component for multiple use cases.
+
+  ListerProfiles
+  ----------
+  A set of pre-configured values for list generation. Define the desired
+  configurations in the `listerProfiles` array using the following options:
+  
+  @option  view  {string}  Name of view type that you want to use for each list item eg: 'teaser' | 'title' etc.
+  @option  list  {string}  Class names for <ul> tag element
+  @option  item  {string}  Class names for <li> tag element
+
+  Example:
+  ```
+  {% set listerProfiles = {
+    teaser: {
+    view: 'teaser',
+    list: 'border-t border-gray-100',
+    item: 'py-2',
+  } %}
+  ```
+
+  Usage Example:
+  ```
+  {% include '_components/_lister' with { params: {
+    entries: collect(indexEntries).all(),
+    kind: 'teaser',
+    class: 'pt-10',
+    list_class: 'mt-10 md:mt-20',
+    list_attributes: {
+      'x-data': "{ activeSlideout: null }",
+    },
+    item_class: 'py-2',
+  } } %}
+  ```
+#}
+
+
+
+{# Parameters #}
+{% set listerEntries = params.entries ?? [] %}
+{% set listerKind = params.kind ?? 'teaser' %}
+{% set listerClass = params.class ?? '' %}
+{% set listerListClass = params.list_class ?? '' %}
+{% set listerListAttr = params.list_attributes ?? {} %}
+{% set listerItemClass = params.item_class ?? '' %}
+
+
+
+{# Config #}
+{% set listerProfiles = {} %}
+
+
+
+{# Variables #}
+{% set listerConfig = listerProfiles[listerKind] ?? listerProfiles.teaser %}
```

Comment:
`listerProfiles.teaser` may not be defined (right now it is an empty array) so we should have a fallback `?? {}`.

Reply:
Done.

Reply:
The `( )` brackets are redundant.